app/controllers/conferences_controller.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
+EVENTS_PER_PAGE = 3
```

Comment:
I don't think we have thought about making it configurable. We decided to hardcode it from the beginning and just go with the number 3, as 3 events look nice on the splash page. 

A URI param may be unnecessary as this variable should rarely change. I do think a configuration file is great because it makes the code cleaner. I'll do some research and send another PR to make some hardcoded parameters (such as `mailbot.rb`) configurable.